docs_2.0/manual_EN/3.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -0,0 +1,203 @@
+```
+
+>**NOTE**: Nebula Graph provides services to the clients through port TCP3699 by default. You can adjust the port number by modifying the [network configurations](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/3.configurations/3.meta-config/#networking_configurations).
+
+## Check the service data and logs
+
+All the data and logs of Nebula Graph are stored persistently in the `nebula-docker-compose/data` and `nebula-docker-compose/logs` directories.
+
+The structure of the directories is as follows:
+
+```text
+nebula-docker-compose/
+  |-- docker-compose.yaml
+  |-- data
+  |     |- meta0
+  |     |- meta1
+  |     |- meta2
+  |     |- storage0
+  |     |- storage1
+  |     `- storage2
+  `-- logs
+        |- meta0
+        |- meta1
+        |- meta2
+        |- storage0
+        |- storage1
+        |- storage2
+        `- graph
+```
+
+## Stop the Nebula Graph services
+
+You can run the following command to stop the Nebula Graph services:
+
+```shell
+$ docker-compose down -v
+```
+
+The following information indicates you have successfully stopped the Nebula Graph services:
+
+```shell
+Stopping nebula-docker-compose_storaged1_1 ... done
+Stopping nebula-docker-compose_storaged0_1 ... done
+Stopping nebula-docker-compose_graphd_1    ... done
+Stopping nebula-docker-compose_storaged2_1 ... done
+Stopping nebula-docker-compose_metad0_1    ... done
+Stopping nebula-docker-compose_metad1_1    ... done
+Stopping nebula-docker-compose_metad2_1    ... done
+Removing nebula-docker-compose_storaged1_1 ... done
+Removing nebula-docker-compose_storaged0_1 ... done
+Removing nebula-docker-compose_graphd_1    ... done
+Removing nebula-docker-compose_storaged2_1 ... done
+Removing nebula-docker-compose_metad0_1    ... done
+Removing nebula-docker-compose_metad1_1    ... done
+Removing nebula-docker-compose_metad2_1    ... done
+Removing network nebula-docker-compose_nebula-net
+```
+
+## Other ways to install Nebula Graph
+
+* [Using Source Code](TODO) (TODO)
+* [Using Docker](TODO) (TODO)
+* [Using .rpm or .deb Files](TODO) (TODO)
+
+## FAQ
+
+### Q1: How to update the nebula-console client
+
+To update the nebula-console client, use the `docker pull` command in the `nebula-docker-compose` directory on your host. For example, if you want to update nebula-console for the Nebula Graph 1.0 series, run the following command.
+
+```Shell
+docker pull vesoft/nebula-console:nightly
+```
+
+If you want to update nebula-console for the Nebula Graph 2.0 pre-release, run the following command instead.
+
+```Shell
+docker pull vesoft/nebula-console:v2-preview-nightly
+```
+
+### Q2: How to update the docker images of Nebula Graph services
+
+To update the images of graphd, storaged, and metad, run `docker-compose pull` in the `nebula-docker-compose` directory.
+
+## What to Do Next
+
+You can start using Nebula Graph by creating spaces and inserting data. For more information, see [CRUD in Nebula Graph](TODO) [TODO].
```

Comment:
spaces --> graph spaces?

You can create a graph space and operate data in Nebula Graph?